### PR TITLE
[WIP] 1st RGSoC Blog Entry 

### PR DESCRIPTION
**Pull request by Karla-Isabel-Sandoval**
Finishing some edits on the About me section-

### CODE REVIEW

**Comment by alicetragedy** [comment]
this looks lovely. just ping me when it's done so I can merge! <3

File: blog/_posts/2015-07-22-hello-we-are-cocoagems.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+I decided to apply because I figured that regardless of the outcome-- applying would be a great learning experience. I had no idea what the odds of getting in would be but I knew that the only way to find out was to try. There is an endless list and reasons as to why I decided to apply. Mainly I love  the idea of learning more about open source and reaching out to potential project mentors to learn about their projects. Reading through their project proposals allowed us to learn a ton just by trying to figure out what each project was about. Our learning speed accelerated before being admitted to RGSoC by the mere act of applying. And that is precisely why I would encourage everyone who is interested in growing their skills through mentorship and Open Source contribution to apply.
+
+
+
+
+<img src="/img/blog/2015/CocoaGemsCoaches.jpg" alt="Team CocoaGems">
```
Comment (mtodd) [inline blog/_posts/2015-07-22-hello-we-are-cocoagems.md:38]: This could be `Team CocoaGems`

**Comment by Karla-Isabel-Sandoval** [comment]
Our final edits are done and we are ready for an official pull request.

Thank you RGSoC!!

**Comment by alicetragedy** [comment]
congratulations on your first pull request to the `summer-of-code` repo, @Karla-Isabel-Sandoval & @emkosz! :confetti_ball: :dancers:

**Comment by mtodd** [comment]
:clap: :sparkles: :fireworks:

**Comment by Karla-Isabel-Sandoval** [comment]
Awww Thanks for your help the other day Matt!
You took care of that question quickly.

On Wed, Jul 22, 2015 at 1:47 PM, Laura < notifications@github.com > wrote:
congratulations on your first pull request the the summer-of-code , @Karla-Isabel-Sandoval & @emkosz !

—
Reply to this email directly or view it on GitHub .

Not using Mixmax yet?

**Pull request merged**